x/community/keeper/incentives_test.go

```diff
@@ -134,9 +139,147 @@ func (suite *IncentivesTestSuite) TestStartDisableInflationUpgrade() {
 	})
 }
 
+func (suite *IncentivesTestSuite) TestStartCommunityFundConsolidation() {
+	suite.SetupTest()
+	ak := suite.App.GetAccountKeeper()
+
+	initialFeePool := distrtypes.FeePool{
+		CommunityPool: sdk.NewDecCoins(
+			sdk.NewDecCoinFromDec("ukava", sdk.NewDecWithPrec(123456, 2)),
+			sdk.NewDecCoinFromDec("usdx", sdk.NewDecWithPrec(654321, 3)),
+		),
+	}
```

Comment:
i think we should add an additional test case where the fee pool is `distrtypes.InitialFeePool()` (zero) as a sanity check this works on fresh chains that will have an empty fee pool.